app/preprints/-components/preprint-node-display/component.ts

```diff
@@ -0,0 +1,48 @@
+import Component from '@glimmer/component';
+import PreprintModel from 'ember-osf-web/models/preprint';
+import { inject as service } from '@ember/service';
+import Intl from 'ember-intl/services/intl';
+import { task } from 'ember-concurrency';
+import { waitFor } from '@ember/test-waiters';
+import Store from '@ember-data/store';
+import { taskFor } from 'ember-concurrency-ts';
+import NodeModel from 'ember-osf-web/models/node';
+
+interface NodeDisplayArgs {
+    preprint: PreprintModel;
+    preprintWord: string;
+}
+
+export default class PreprintNodeDisplay extends Component<NodeDisplayArgs> {
+    @service intl!: Intl;
+    @service store!: Store;
+
+    preprint = this.args.preprint;
+    node!: NodeModel;
+    nodeId!: string;
```

Comment:
These are both possibly `null` if the preprint doesn't have a supplemental node, right?
```suggestion
    node?: NodeModel;
    nodeId?: string;
```